cmd/provide.go

```diff
@@ -10,13 +10,16 @@ import (
 
 var provideModuleCmdSrc string
 var provideModuleCmdDesc string
+var provideModuleCmdExtensions []string
 
 // init - inits flags of provide module command
 func init() {
 	provideModuleCmd.Flags().StringVarP(&provideModuleCmdSrc, "source", "s",
 		"", "the path to the MTA project; the current path is set as the default")
 	provideModuleCmd.Flags().StringVarP(&provideModuleCmdDesc, "desc", "d", "",
 		`the MTA descriptor; supported values: "dev" (development descriptor, default value) and "dep" (deployment descriptor)`)
+	provideModuleCmd.Flags().StringSliceVarP(&provideModuleCmdExtensions, "extensions", "e", nil,
```

Comment:
I think it won't affect the result because you can't add or remove modules in the extension files, but it seems too implementation-specific to count on that so I prefer to pass it to the command just in case (like I pass it to all other commands that parse the mta.yaml).